integration-tests/src/test/java/oracle/weblogic/kubernetes/ItParameterizedDomain.java

```diff
@@ -444,6 +450,172 @@ public void testLivenessProbe() {
         .isTrue();
   }
 
+  @Test
+  @DisplayName("Test dataHome override in a domain with domain in image type")
+  public void testDataHomeOverrideDomainInImage() {
+    // in this domain, set dataHome to /u01/oracle/mydata in domain custom resource
+    // the domain contains JMS and File Store configuration
+    // File store directory is set to /u01/oracle/customFileStore/ in the model file which should be overridden
+    // by dataHome
+    // File store and JMS server are target to cluster-1
```

Comment:
File store and JMS server are targeted to the WebLogic cluster cluster-1

Reply:
fixed